Grievance_Backend/grievance/src/main/java/com/grievance/service/EmployeeServiceImpl.java

```diff
@@ -47,7 +48,7 @@ public class EmployeeServiceImpl implements EmployeeService {
   public Optional<EmployeeOutDto> saveEmployee(
     final EmployeeInDto employeeInDto) {
     if (employeeRepository.findByEmail(employeeInDto.getEmail()) != null) {
-      return Optional.empty();
+      throw new EmployeeAlreadyExistException(employeeInDto.getEmail());
     }
     Employee employee = converToEntity(employeeInDto);
```

Comment:
change method name to  convertToEntity.

Reply:
Okay, I'll proceed with renaming the method to "convertToEntity" in the code.